BikeRental.tests/RentalTests.cs

```diff
@@ -0,0 +1,148 @@
+﻿using BikeRental.Domain.Enums;
+using BikeRental.tests;
+
+namespace BikeRental.Tests;
+
+public class RentalTests : IClassFixture<RentalFixture>
+{
+    private readonly RentalFixture _fixture;
+
+    public RentalTests(RentalFixture fixture)
+    {
+        _fixture = fixture;
+    }
+
+    /// <summary>
+    /// Checks that all models of type Sport are correctly found.
+    /// </summary>
+    [Fact]
+    public void ShouldFindAllSportBikes()
+    {
+        List<string> expected =  ["SpeedMax", "TurboSport" ];
+
+            var actual = _fixture.Models
+                .Where(m => m.Type == BikeType.Sport)
+                .Select(m => m.Name)
+                .OrderBy(n => n)
+                .ToList();
+
+            Assert.Equal(expected, actual);
+    }
+
+    /// <summary>
+    /// Checks the top 5 models by total profit and their order.
+    /// </summary>
+    [Fact]
+    public void ShouldCalculateTop5ModelsByProfit()
+    {
+        List<string> expected =
+        [
+            "TurboSport",
+            "VoltBike",
+            "Mountain Pro",
+            "EcoBike",
+            "Urban 2000"
+        ];
+
+        var actual = _fixture.Rentals
+            .GroupBy(r => r.Bike.Model)
+            .Select(g => new
+            {
+                Name = g.Key.Name,
```

Comment:
Можно пропустить именование так как оно уже называется Name
```suggestion
                g.Key.Name,
```